modules/temp-channels/events/interactionCreate.js

```diff
@@ -0,0 +1,108 @@
+const {MessageActionRow, Modal, TextInputComponent } = require('discord.js');
+const {usersList, channelMode, userAdd, userRemove, channelEdit} = require('../channel-settings');
+const {localize} = require('../../../src/functions/localize');
+const {Op} = require('sequelize');
+
+module.exports.run = async function (client, interaction) {
+    if (!client.botReadyAt) return;
+    if (interaction.guild.id !== client.config.guildID) return;
+    if (interaction.isButton()) {
+        const vc = await client.models['temp-channels']['TempChannel'].findOne({
+            where: {
+                [Op.and]: [
+                    {id: interaction.member.voice.channelId},
+                    {creatorID: interaction.member.id}
+                ]
+            }
+        });
+
+        if (!vc) {
+            interaction.reply({ ephemeral: true,
+                content: interaction.client.configurations['temp-channels']['config']['notInChannel']});
+            return;
+        }
```

Comment:
Please keep in mind that not all interactions are part of temp-channels and might be from other modules. Please verify that you only respond to interactions that are part of temp-channels